<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="gameLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>331</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:700;&quot;&gt;CHESS&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;by&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Sanjay Gounder,&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Tate Reynolds,&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Gage Buss,&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Thatcher Geary &amp;amp; Jonas Knochelmann&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tutorialButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>340</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Tutorial</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="muteButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mute Music</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
